benchmark/child_process/child-process-exec-stdout.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common.js');
+const bench = common.createBenchmark(main, {
+  len: [64, 256, 1024, 4096, 32768],
+  dur: [5]
+});
+
+const exec = require('child_process').exec;
+function main(conf) {
+  bench.start();
+
+  const dur = +conf.dur;
+  const len = +conf.len;
+
+  const msg = '"' + Array(len).join('.') + '"';
```

Comment:
OK, done! The benchmark shows frighteningly similar results for Node.js 6.2.0 vs. this PR.

6.2.0:

```
$ node benchmark/child_process/child-process-exec-stdout.js 
child_process/child-process-exec-stdout.js len=64 dur=5: 40873.45059
child_process/child-process-exec-stdout.js len=256 dur=5: 40853.93567
child_process/child-process-exec-stdout.js len=1024 dur=5: 40878.88575
child_process/child-process-exec-stdout.js len=4096 dur=5: 40851.36544
child_process/child-process-exec-stdout.js len=32768 dur=5: 40876.38241
$
```

This PR:

```
$ ./node benchmark/child_process/child-process-exec-stdout.js 
child_process/child-process-exec-stdout.js len=64 dur=5: 40873.08798
child_process/child-process-exec-stdout.js len=256 dur=5: 40860.52815
child_process/child-process-exec-stdout.js len=1024 dur=5: 40878.13205
child_process/child-process-exec-stdout.js len=4096 dur=5: 40869.97254
child_process/child-process-exec-stdout.js len=32768 dur=5: 40863.11006
$
```